src/pybind/mgr/orchestrator/module.py

```diff
@@ -1357,6 +1371,10 @@ def self_test(self) -> None:
         self._set_backend('')
         assert self._select_orchestrator() is None
         self._set_backend(old_orch)
+        self._set_fail_fs(True)
+        assert self._get_fail_fs_value() is True
+        self._set_fail_fs(False)
+        assert self._get_fail_fs_value() is False
```

Comment:
it might not matter but can we have it preserve the old `fail_fs` value the same way it is done for the backend. Right now it will always leave the function with it set false.

Reply:
Right, I'll make the change and push. Thanks!

Reply:
Done